docs/source/getting-started/assessing-utility/accuracy.ipynb

```diff
@@ -0,0 +1,374 @@
+      "text/plain": [
+       "<Figure size 1000x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Join dataframes. \n",
+    "all_df = dp_values.join(actual_values, on=['YEAR', 'QUARTER'])\n",
+    "all_df = all_df.with_columns(\n",
+    "    pl.format(\"{}-Q{}\", pl.col(\"YEAR\"), pl.col(\"QUARTER\")).alias(\"Year-Quarter\")\n",
+    ")\n",
+    "\n",
+    "# Convert to date time format for line plot. \n",
+    "all_df = all_df.with_columns(\n",
+    "    pl.col(\"Year-Quarter\").map_elements(lambda x: np.datetime64(pd.Period(x, freq='Q').to_timestamp())).alias(\"Date\")\n",
+    ")\n",
+    "\n",
+    "# Plotting.\n",
+    "dates = all_df.select(pl.col(\"Date\")).to_numpy().flatten()\n",
+    "actual_vals = all_df.select(pl.col(\"Actual\")).to_numpy().flatten()\n",
+    "dp_vals = all_df.select(pl.col(\"DP\")).to_numpy().flatten()\n",
+    "\n",
+    "\n",
+    "plt.figure(figsize=(10, 6))\n",
+    "plt.plot(dates, actual_vals, marker='o', label=\"Actual\")\n",
+    "plt.plot(dates, dp_vals, marker='o', label=\"DP\")\n",
+    "plt.title('Counts Across Years and Quarters')\n",
+    "plt.xlabel('Date')\n",
+    "plt.ylabel('Actual')\n",
+    "plt.grid(True)\n",
+    "plt.legend()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It appears the differentially private values can vary significantly higher or lower for particular quarter and year combinations. However, on average, the differentially private values follow a similar trend as the actual values. \n",
+    "\n",
+    "To evaluate this using the `accuracy,` method, apply it to the naive query itself before you apply `release()` and `collect` on it. \n",
+    "\n",
+    "The `accuracy` method has one optional parameter: alpha, which is the optional statistical significance to use to compute accuracy. If you don't specify alpha, you'll still get a table, but it won't have an accuracy value. In this case, we will specify alpha to be 0.05. "
```

Comment:
- We should explain what alpha is before just showing the code and talking about it here. Wouldn't make sense to the reader otherwise. Also, we should explain what this notion of accuracy is in a slightly mathematical way.
- We should also specify what the input/output are for this accuracy function.

Reply:
i added the words input and output to make it clear. i understand alpha as just a statistical signif in this case. is it something different?